docs/components/early-access/alpha/sap/camunda-sap-integration.md

```diff
@@ -21,6 +21,7 @@ These features run in the customer's [SAP BTP instance](https://www.sap.com/prod
 
 All SAP integration artifacts have the following technical requirements:
 
+- Camunda version 8.5+
```

Comment:
With this, I just want to ensure this shouldn't be backported to `8.5`, and should remain in early access for only this version of docs?

Reply:
for now, it should remain in "early access" - but yes, for the GA in April, we need to backport this to 8.5+